CHANGELOG.md

```diff
@@ -27,6 +27,8 @@
   - more detailed error messages in some scenarios
   - some errors now cause the script to pause, instead of crashing the game
 - SCM functions **(0AB1)** now keep their own GOSUB's call stack
+- opcode **0A8D (read_memory)** now allows size argument in range 0-4
+- new opcode **0B1E ([sign_extend](https://library.sannybuilder.com/#/sa/bitwise/0B1E))**
```

Comment:
please move under 'new and updated opcodes' section

Reply:
fixed